director/director.go

```diff
@@ -281,7 +281,11 @@ func redirectToCache(ginCtx *gin.Context) {
 
 	linkHeader := ""
 	first := true
-	for idx, ad := range cacheAds {
+	cachesToSend := 6
```

Comment:
Do we want this to be a top-level constant? 
Also, can you add a brief comment explaining this magic number? i.e. what's our intent to limit caches to 6